llvm/test/tools/llvm-symbolizer/approximate-line-generated.s

```diff
@@ -0,0 +1,153 @@
+# REQUIRES: x86-registered-target
```

Comment:
Can this test be ported to use the elaborated test infrastructure too?

Reply:
Done